src/js/event-target.js

```diff
@@ -1,12 +1,50 @@
 /**
- * @file event-target.js
+ * @file src/js/event-target.js
  */
 import * as Events from './utils/events.js';
 
+/**
+ * An EventTarget class that seeks to emulate much of the builtin
+ * HTML EventTarget functionalify as well as add all kinds of ease
+ * of use funcionality
+ *
+ * @class EventTarget
+ */
 const EventTarget = function() {};
 
+/**
+ * A list of events that the EventTarget is allowed to use
+ *
+ * @property EventTarget.prototype.allowedEvents_
+ * @private
+ */
 EventTarget.prototype.allowedEvents_ = {};
 
+/**
+ * This function is a cross browser wrapper around addEventListener
+ * with a much nicer syntax
```

Comment:
I think it'd be better to describe what this does rather than its implementation, the same way the `off` method does.